### PR TITLE
feat(MINOR): add auto changelog action workflow

### PR DESCRIPTION
**Pull request by Orkuncakilkaya**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -37,31 +37,28 @@ jobs:
...
+          java-version: '11'
+
+      - uses: shivammathur/setup-php@v2
         with:
-          commit_message: Bump version to ${{ steps.semver.outputs.version }}
-          skip_fetch: true
-      - name: Create Release
-        id: release
-        uses: actions/create-release@v1
+          php-version: "${{ matrix.php_version }}"
```
Comment (ilfa) [inline .github/workflows/release.yml:50]: Are you sure about matrix for release?

File: release/.releaserc.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
+module.exports = {
+    "branches": [
+        "main",
+        {
+            "name": "test",
```
Comment (ilfa) [inline release/.releaserc.js:5]: You have only `develop` and `main` branches in your workflow, do you want to change the workflow?

**Review by ilfa — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (Orkuncakilkaya) [reply .github/workflows/release.yml:50]: Should I only test on single version or remove the test altogether?

File: release/.releaserc.js (hunk 2 above)
Comment (Orkuncakilkaya) [reply release/.releaserc.js]: I changed the branches, can you check again?

File: .github/workflows/release.yml (hunk 1 above)
Comment (ilfa) [reply .github/workflows/release.yml:50]: It is a release workflow and looks like there are no tests run here

Comment (Orkuncakilkaya) [reply .github/workflows/release.yml]: I removed test job from release, and used php version 7.3 as single version

File: release/.releaserc.js (hunk 2 above)
Comment (Orkuncakilkaya) [reply release/.releaserc.js]: Added develop branch for prereleases

**Review by ilfa — APPROVED** [review]

**Pull request merged**